index.src.html

```diff
@@ -803,10 +807,11 @@ <h3 id="add-report" algorithm>
 
   3. Append |r| to |observer|'s <a>report queue</a>.
 
-  4. If the size of |observer|'s <a>report queue</a> is 1, <a>Queue a task</a>
-     to [[#invoke-observers]] with a copy of the <a>registered reporting
-     observer list</a> of the <a spec=ecmascript lt=realm>ECMAScript global
-     environment</a> associated with |observer|.
+  4. If the size of |observer|'s <a>report queue</a> is 1:
+     1.  Let |global| be the <a spec=ecmascript>global object</a> of the <a
+         spec=ecmascript>realm</a> associated with |observer|.
+     2.  <a>Queue a task</a> to [[#invoke-observers]] with a copy of |global|'s
+         <a>registered reporting observer list</a>.
```

Comment:
Drive-by preexisting issue: These days we don't like bare "Queue a task". This should be "Queue a global task on the XXX task source given |global| to ...".

Reply:
Thanks -- filed #220.